app/services/facebook_service.rb

```diff
@@ -0,0 +1,13 @@
+class FacebookService
+  def initialize(access_token)
+    @access_token = access_token
+  end
+
+  def profile
+    client.get_object('me', fields:'email, name, first_name, picture.type(large)')
+  end
+
+  def client
```

Comment:
This client should be private to avoid something like `FacebookService.new().client`